### DISCUSSION TITLE
How it work setVideoRef?

### DISCUSSION THREAD

**Discussion by achilan** [ValentinH/react-easy-crop#386] [General]
I used this property with a video ref

**Comment by ValentinH** [comment]
Here's an example: https://codesandbox.io/s/react-easy-crop-forked-6rtj6b?file=/src/index.js

**Reply by achilan** [reply]
U have example when I resize video after cropping?

**Reply by ValentinH** [reply]
This is much more complex than cropping images. I don't think this is possible in the browser. You need to do this server side with a tool like ffmpeg. There might be some SaaS services that provides API to do this though.
